spec/decorators/member_decorator_spec.rb

```diff
@@ -1,4 +1,30 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
+require 'draper/test/rspec_integration'
 
 RSpec.describe MemberDecorator do
+  describe '#hours_worked_in_the_three_last_months' do
+    let(:generate_member_decorator) { described_class.new(build_stubbed(:member)) }
+
+    context 'when active admin view is generated' do
```

Comment:
I think `'when no options are given'` would be a more accurate context, as the MemberDecorator is not aware of the existence of ActiveAdmin. You could for example use it in a regular view.

Reply:
done